udapi/block/read/conllu.py

```diff
@@ -1,22 +1,17 @@
-import logging
 import re
-import bz2
 
 from udapi.core.basereader import BaseReader
 from udapi.core.root import Root
 
+# Compile a set of regular expressions that will be searched over the lines.
+RE_SENT_ID = re.compile(r'^# sent_id\s*=\s+(\S+)')
```

Comment:
The equal sign was added in UD v2.0 [specification](http://universaldependencies.org/format.html).
Now, I've made the equal sign optional in the reader to support pre-v2 files.